src/javascripts/helpers/validationForm.js

```diff
@@ -64,8 +64,15 @@ export const validateTransaction = (data) => {
     },
   ]
 
-  for(const key in data){
-    const value = data[key];
+  const transactionCheck = {
+    category: data.category,
+    date: data.date,
+    outflow: data.outflow,
+    note: data.note
+  }
+
+  for(const key in transactionCheck){
+    const value = transactionCheck[key];
```

Comment:
What do value use for?

Reply:
The value is used to assign each property of the object transactionCheck. These properties of the object transactionCheck are the fields that I want to validate.

Reply:
Got it